### PR TITLE
Change versioning description

### PR DESCRIPTION
**Pull request by krzysztofwolski**
Wanted to suggest that saved versions are tested ones and updating can break env.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -42,8 +42,8 @@ All Saleor services started from a single repository
 
 
 ## How to update the subprojects into newest versions?
-By default, each of the subprojects is bound to particular commit.
-In order to update all of them to their newest master versions, run `git submodule update --remote`
+By default, this repository contains newest current (stable) versions. When new release appear, pull new version of this repository.
```
Comment (tomaszszymanski129) [inline README.md]: It's a bit sketchy, due to some reasons this doesn't contain stable versions for now (dev dockerfiles added after last stable for dashboard and storefront). I've created a reminder issue in this repo to update subprojects when we'll release newest stable. Not sure if we should merge this part for now.

Comment (krzysztofwolski) [reply README.md]: @maarcingebala when do we plan to announce new version?

Comment (krzysztofwolski) [reply README.md]: Alternative solution would be creating patches for a last stable version and just bump minor version number.

Comment (maarcingebala) [reply README.md]: We don't know the release date yet.

**Pull request merged**